sdk/samples/iot/iot_sample_common.h

```diff
@@ -18,42 +18,65 @@
 //
 // Logging
 //
-#define IOT_SAMPLE_LOG_ERROR(...)                                                  \
-  do                                                                               \
-  {                                                                                \
+#define IOT_SAMPLE_LOG_ERROR(...) \

```

Comment:
See my previous comment on the naming of these macros

Reply:
@momuno - you came up with this naming prefix recently. Should I keep `IOT_SAMPLE`, or make it something else?

Reply:
I contacted Mollie, we will likely follow up outside of this PR.

Reply:
See my earlier note.  These are used across samples based on the iot_sample_common files and meant to be distinguished from static functions belonging to each sample.  Additionally, it is meant to show that these are custom functions found in io_sample_common and not part of the AZ API.